src/m365/spo/commands/web/web-installedlanguage-list.ts

```diff
@@ -0,0 +1,69 @@
+import { Logger } from '../../../../cli';
+import { CommandOption } from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+import { WebInstalledLanguagePropertiesCollection } from './WebPropertiesCollection';
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  webUrl: string;
+}
+
+class SpoWebInstallEdlanguageListCommand extends SpoCommand {
```

Comment:
Let's add `installed` to the list of known words in https://github.com/pnp/cli-microsoft365/blob/642517813d550252db3115cb2534f8fe7e17d98f/.eslintrc.js#L3 so that we can have proper casing